### DISCUSSION TITLE
unable to install chatterbot using pip

### DISCUSSION THREAD

**Discussion by shivabasayya007** [explosion/spaCy#13326] [Help: Coding & Implementations] [answered]
it gives an error stating : **note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error**

**Comment by svlandeg** [comment, marked as answer]
Hi! We don't maintain that tool, the original repo is here: https://github.com/gunthercox/ChatterBot. It's probably best to ask your question there.
